### PR TITLE
TSE: signature verifier helper script

### PR DESCRIPTION
**Pull request by mhcp6**

### CODE REVIEW

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import ecdsa
+from hashlib import sha384, sha256
+import base64
+from dateutil import parser
+from datetime import timezone
+import pprint
+from asn1crypto.core import Integer, ObjectIdentifier, Sequence, OctetString, PrintableString, Any
```
Comment (TheJJ) [inline stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:7]: asn1crypto as dependency?

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    public_key = qr[11]
+    if qr[8] != "ecdsa-plain-SHA384":
+        raise NotImplemented
+    else:
+        signaturealgorithm = SignatureAlgorithm_seq()
+        signaturealgorithm["signatureAlgorithm"] = "0.4.0.127.0.7.1.1.4.1.4"  # ecdsa-plain-SHA384
+        vk = ecdsa.VerifyingKey.from_string(base64.b64decode(public_key), curve=ecdsa.BRAINPOOLP384r1, hashfunc=sha384)
+
+    if qr[9] != "unixTime":  # case sensitive != UnixTime ...
```
Comment (TheJJ) [inline stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:67]: what's the difference :D

**Review by TheJJ — CHANGES_REQUESTED** [review]

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py (hunk 1 above)
Comment (mhcp6) [reply stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:7]: do you want to build DER objects by hand?

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py (hunk 2 above)
Comment (mhcp6) [reply stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:67]: 'u' != 'U'

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py (hunk 1 above)
Comment (TheJJ) [reply stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:7]: it's missing in the pyproject.toml :)

File: stustapay/tse/diebold_nixdorf_usb/tse_verifier.py (hunk 2 above)
Comment (TheJJ) [reply stustapay/tse/diebold_nixdorf_usb/tse_verifier.py:67]: qrcode has to have `unixTime`, but the DN export returns `UnixTime`...

**Review by TheJJ — APPROVED** [review]

**Pull request merged**